### PR TITLE
Feature/db jooq

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
Add whitelists for ERC20 tokens and wallets with jOOQ DB layer.

### CODE REVIEW

File: deploy/db/sql/populate.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-- Populate wallets table
+INSERT INTO notary.wallets (wallet, irohausername)
+VALUES ('0x00aa39d30f0d20ff03a22ccfc30b7efbfca597c2', 'user1@notary');
```
Comment (x3medima17) [inline deploy/db/sql/populate.sql]: Maybe it is better to store all these addresses in a separate config file?

File: src/test/integration/EthTokensProviderTest.kt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * Test Postgres ethereum ERC20 tokens provider
+ */
+class EthTokensProviderTest {
+
+    /**
+     * List all tokens in database
+     */
+    //    @Test
```
Comment (x3medima17) [inline src/test/integration/EthTokensProviderTest.kt]: Looks like it is not a test

File: src/test/integration/EthTokensProviderTest.kt [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        )
+
+        DSL.using(connection).use { ctx ->
+            val tokens = Tokens.TOKENS
+
+            // read
+            ctx.select(tokens.WALLET, tokens.TOKEN)
+                .from(tokens)
+                .forEach {
+                    println(it)
```
Comment (x3medima17) [inline src/test/integration/EthTokensProviderTest.kt]: No need in println in tests

File: src/test/integration/EthWalletsProviderTest.kt [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * Test Postgres ethereum wallets provider
+ */
+class EthWalletsProviderTest {
+
+    /**
+     * List all wallets in database
+     */
+    //    @Test
```
Comment (x3medima17) [inline src/test/integration/EthWalletsProviderTest.kt]: sab

File: src/test/integration/EthWalletsProviderTest.kt [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        )
+
+        DSL.using(connection).use { ctx ->
+            val wallets = Wallets.WALLETS
+
+            // read
+            ctx.select(wallets.WALLET, wallets.IROHAUSERNAME)
+                .from(wallets)
+                .forEach {
+                    println(it)
```
Comment (x3medima17) [inline src/test/integration/EthWalletsProviderTest.kt]: sab

File: deploy/db/sql/populate.sql (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply deploy/db/sql/populate.sql]: It is testing data, not configuration. These data is used for manual and integration testing.

File: src/test/integration/EthTokensProviderTest.kt (hunk 2 above)
Comment (Alexey-N-Chernyshov) [reply src/test/integration/EthTokensProviderTest.kt]: It's a way to disable test in CI, but run it manually.

**Pull request merged**